backend/internal/supabase/migrations/20240917002108_base_schema.sql

```diff
@@ -60,4 +60,6 @@ create table if not exists review (
     comment text,
     created_at timestamp with time zone default now(),
     updated_at timestamp with time zone default now()
+    -- btw if query fails due to unique constraint, id will still be incremented
+    add constraint unique_user_media unique (user_id, media_id, media_type);
```

Comment:
since this base_schema migration script was already run in supabase, it'll yell at us for trying to run the same script again - instead, let's move the constraint addition to its own migration file 

to make the new migration file:
* easy one-time solution: just make a `20240921002108_add_unique_review_constraint.sql` file in `/migrations`
* proper and more helpful way is using the supabase CLI - [download it](https://supabase.com/docs/guides/cli/getting-started), cd into `backend/internal`, and try running `supabase migration new add_unique_review_constraint`. using the cli lets you spin up a local instance of the db, verify that your scripts run without errors, and see your data via a locally running dashboard, but tbh i haven't fully tested it on another machine so idk if it'll work for you guys without more setup steps. if you're feeling adventurous, you can give it a shot [(here are the docs](https://supabase.com/docs/guides/cli/local-development?queryGroups=access-method&access-method=postgres#database-migrations)) and i can help!

and this new file should just have
```
alter table review add constraint unique_user_media unique (user_id, media_id, media_type);
-- btw if query fails due to unique constraint, review.id will still be incremented
```